docs/content/latest/faq/compatibility.md

```diff
@@ -42,6 +42,21 @@ As highlighted in [Distributed PostgreSQL on a Google Spanner Architecture – Q
 
 YugabyteDB's goal is to remain as compatible with PostgreSQL as possible. If you see a feature currently missing, please file a [GitHub issue](https://github.com/yugabyte/yugabyte-db/issues) for us.
 
+## Are YSQL and YCQL compatible with each other?
```

Comment:
Break this question into 2 questions as shown below.

## Can I insert data using YCQL but read using YSQL or vice versa?

The YugabyteDB APIs are currently isolated and independent from one another. Data inserted or managed by one API cannot be queried by the other API. Additionally, there is no common way to access data across the APIs. An external framework, such as Presto, might be useful for simple use cases. For an example that joins YCQL and YSQL data, see the blog post about [Presto on YugabyteDB: Interactive OLAP SQL Queries Made Easy](https://blog.yugabyte.com/presto-on-yugabyte-db-interactive-olap-sql-queries-made-easy-facebook/).

Allowing YCQL tables to be accessed from the PostgreSQL-compatible YSQL API as foreign tables using foreign data wrappers (FDW) is on the roadmap. You can comment and/or increase the priority of the associated [GitHub](https://github.com/yugabyte/yugabyte-db/issues/830) issue.

## When should I pick YCQL over YSQL?

You should pick YCQL over YSQL if your application:

    * Does not require fully-relational data modeling constructs such as foreign keys and JOINs. Note that strongly-consistent secondary indexes and unique constraits are supported by YCQL.
    * Requires storing large amounts of data (say 10TB or more).
    * Needs to serve low-latency (sub-ms) queries.
    * Needs TTL-driven automatic data expiry.
    * Needs to integrate with stream processors such as Apache Spark and KSQL.

If you have a specific use case in mind, share it in our [Slack community](https://www.yugabyte.com/slack) and the community can help you decide the best approach.

Reply:
Split and edited.